Eatery/Controllers/Eateries/Campus/CampusEateriesViewController.swift

```diff
@@ -14,6 +14,16 @@ import UIKit
 
 class CampusEateriesViewController: EateriesViewController {
 
+    
```

Comment:
let's try to keep all the new line spacing consistent -- for most cases, this should just be one line